packages/rpc-core/src/rpc-methods/getBlockCommitment.ts

```diff
@@ -0,0 +1,13 @@
+import { Slot, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+type GetBlockCommitmentApiResponse = Readonly<{
+    commitment: U64UnsafeBeyond2Pow53Minus1[] | null;
```

Comment:
The awful thing about this is that it will never, ever be right with the current RPC. These numbers are way larger than JS numbers.

```shell
 % curl https://api.mainnet-beta.solana.com -X POST -H "Content-Type: application/json" -d '                                                                                                   
  {
    "jsonrpc": "2.0", "id": 1,
    "method": "getBlockCommitment",
    "params":[208151977]
  }
'
{"jsonrpc":"2.0","result":{"commitment":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,382219010785520358],"totalStake":383784289656803160},"id":1}
```

```
> console.log(382219010785520358)
382219010785520400
> console.log(383784289656803160)
383784289656803140
```

We need to deprecate this API as soon as possible and do https://github.com/solana-labs/solana/issues/30741.

Reply:
Related: https://github.com/solana-labs/solana/issues/32662.